Gogogate2 0.1.1

dlbroadfoot opened the pull request:
## Description:
Update to use latest version of gogogate2 component. Expose device temperature as attribute

## Checklist:
  - [ ] The code change is tested and works locally.
  - [ ] Local tests pass with `tox`. **Your PR cannot be merged unless tests pass**

If user exposed functionality or configuration variables are added/changed:
  - [ ] Documentation added/updated in [home-assistant.github.io](https://github.com/home-assistant/home-assistant.github.io)

dlbroadfoot committed:
Gogogate2 - bump version

Uses latest version of library which ensures commands to device are idempotent

dlbroadfoot committed:
Update requirements_all

dlbroadfoot committed:
Expose sensor temperature

dlbroadfoot committed:
update version

dlbroadfoot committed:
import attribute

syssi commented:
Please fix the build error!

dlbroadfoot committed:
Set temperature

balloob commented on homeassistant/components/cover/gogogate2.py:
```diff
@@ -92,6 +93,13 @@ def supported_features(self):
+    def state_attributes(self):
+        """Return the state attributes."""
+        data = super().state_attributes
+        data[ATTR_TEMPERATURE] = self._temperature
```
This should not be added as an attribute but instead be made into it's own sensor. This means that you will need to greate a gogogate2 component to handle the connection and then setup the cover and sensor platforms.

dlbroadfoot replied:
Fair enough. For now, I'll update the PR to get the bug fix in and look at adding the component/sensors later.

For additional data points such as battery_level of the garage tilt sensor - should that be added as an attribute of the cover platform or should it be its own sensor too?

dlbroadfoot committed:
Remove temperature attribute

Removed temperature attribute until it can be re-implemented as a separate sensor.

fabaff reviewed (approved):
Thanks :bird:

fabaff committed:
Update ordering

fabaff committed:
Fix copy-&-paste issue
